src/libvncserver/websockets.c

```diff
@@ -289,13 +291,15 @@ webSocketsHandshake(rfbClientPtr cl, char *scheme)
         return FALSE;
     }
 
-    if ((protocol) && (strstr(protocol, "base64"))) {
+    proto_binary = strstr(protocol, "binary");
```

Comment:
I think that's UB if `protocol` is `NULL`. Maybe just inline those calls?

Reply:
Added a NULL check.